client/starwhale/core/model/default_handler.py

```diff
@@ -39,6 +39,6 @@ def ppl(context: Context) -> None:
     _invoke(context=context, func="_starwhale_internal_run_ppl")
 
 
-@step(needs=["ppl"])
+@step(needs=["ppl"], overwriteable=False)
```

Comment:
`overwriteable` -> `overridable`  or `overwritable`?

Reply:
done but this param is removed as it is confusing